chia/full_node/mempool_manager.py

```diff
@@ -134,7 +134,7 @@ def __init__(
         self.mempool: Mempool = Mempool(
             self.mempool_max_total_cost,
             uint64(self.nonzero_fee_minimum_fpc),
-            uint64(self.constants.MAX_BLOCK_COST_CLVM),
+            uint64(self.max_block_clvm_cost),
```

Comment:
this is a bit gnarly. by making this depend on the fill rate, you also make the fee estimator depend on the fill rate. i.e. a node with a fill rate of 1 will give much lower estimates than a node with fill rate 0.5.
In a way it makes it more accurate now, assuming no farmers change this constant

Reply:
Well, if the estimator doesn't have the true size for max mempool size and max block size, that can make a lot of trouble (Hmm, even more trouble if the mempool doesn't have the correct size).

Reply:
yeah, this rate limit is probably close enough to a constant though. I think the point I was trying to make is that what the fee estimator *really* is interested in is the next winning farmer's block cost limit. And this is probably a good estimate for that (until we bump the rate limit to 1)